engine/gamesys/src/gamesys/resources/res_font.cpp

```diff
@@ -152,7 +152,7 @@ namespace dmGameSystem
         uint8_t* data = (uint8_t*)glyph_bank->m_GlyphData.m_Data;
         uint8_t* glyph_data = GetPointer(data, glyph->m_GlyphDataOffset);
 
-        *out_size = glyph->m_GlyphDataSize;
+        *out_size = glyph->m_GlyphDataSize - 1;
```

Comment:
Hm, why is this change needed? Maybe add a comment about it?

Reply:
It was never accounted for, but the Deflate function handled it anyways.
I'll update the PR